rocketmq-console/src/test/java/org/apache/rocketmq/console/controller/BaseControllerTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.rocketmq.console.controller;
+
+import org.apache.rocketmq.console.BaseTest;
+import org.apache.rocketmq.console.config.RMQConfigure;
+import org.apache.rocketmq.console.support.GlobalExceptionHandler;
+import org.apache.rocketmq.console.support.GlobalRestfulResponseBodyAdvice;
+import org.apache.rocketmq.console.util.MyPrintingResultHandler;
+import org.apache.rocketmq.tools.admin.MQAdminExt;
+import org.junit.Before;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+public abstract class BaseControllerTest extends BaseTest {
+
+    protected MockMvc mockMvc;
+
+    @Mock
+    protected MQAdminExt mqAdminExt;
+
+    @Mock
+    protected RMQConfigure rMQConfigure;
+
+    protected abstract Object getTestController();
+
+    @Before
+    public void beforeInit() throws Exception {
+        MockitoAnnotations.initMocks(this);
+        autoInjection();
+        createMockMvc();
+    }
+
+    protected void mockRmqConfigure() {
+        when(rMQConfigure.getAccessKey()).thenReturn("12345678");
```

Comment:
It seems that this function can be deleted, it have no connection with the tests

Reply:
![image](https://user-images.githubusercontent.com/18254437/122489594-99028500-d012-11eb-8db1-4523b5ffcc4c.png)
If the unit test wants to cover all rows of this method, it need to mock the values of accessKey and secretKey not to be null